cli/release.go

```diff
@@ -14,19 +15,21 @@ import (
 	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
 	"github.com/flynn/flynn/controller/client"
 	ct "github.com/flynn/flynn/controller/types"
+	"github.com/flynn/flynn/pinkerton/registry"
 )
 
 func init() {
 	register("release", runRelease, `
 usage: flynn release
-       flynn release add [-t <type>] [-f <file>] <uri>
+       flynn release add [-t <type>] [-f <file>] [--validate=<validate>] <uri>
        flynn release show [<id>]
 
 Manage app releases.
 
 Options:
-	-t <type>          type of the release. Currently only 'docker' is supported. [default: docker]
-	-f, --file=<file>  release configuration file
+	-t <type>              type of the release. Currently only 'docker' is supported. [default: docker]
+	-f, --file=<file>      release configuration file
+	--validate=<validate>  should the image URI be validated? [default: true]
```

Comment:
This should be `--no-validate` and a boolean, so no `=<validate>` is required. (will be available in `args.Bool`)